packages/cms/src/components/sections/Hero.jsx

```diff
@@ -210,10 +210,12 @@ class Hero extends Component {
 
     // heading & subhead(s)
     const heading = <Fragment>
-      <Parse El="h1" id={contents ? contents.slug : `${stripHTML(profile.title)}-hero`} className="cp-section-heading cp-hero-heading u-font-xxl">
-        {title}
-      </Parse>
-      {subtitleContent}
+      <div className="cp-hero-heading-wrapper">
+        <Parse El="h1" id={contents ? contents.slug : `${stripHTML(profile.title)}-hero`} className="cp-section-heading cp-hero-heading u-font-xxl">
+          {title}
+        </Parse>
+        {subtitleContent}
+      </div>
     </Fragment>;
```

Comment:
@cnavarreteliz I meant for the `<div>` to replace the `<Fragment>`, not be nested inside of it 😬. This `<Fragment>` now serves no purpose, and is unnecessary overhead!